orchagent/copporch.cpp

```diff
@@ -75,12 +77,68 @@ std::map<string, sai_packet_action_t> packet_action_map = {
     {"transit", SAI_PACKET_ACTION_TRANSIT}
 };
 
+std::vector<sai_hostif_trap_id_t> default_trap_ids = {
+    SAI_HOSTIF_TRAP_ID_TTL_ERROR
+};
+
+const string default_trap_group = "default";
```

Comment:
some of other trap ids which are part of trap group may also be trapped, but for us its important to make sure TTL error will be trapped always and on all vendor platforms.